### ISSUE TITLE
bump grammers for fix factorization issue

### ISSUE THREAD

**Issue by andrew-ld** [cher-nov/cryptg#31]
https://github.com/Lonami/grammers/issues/220

**Comment by cher-nov** [comment]
@andrew-ld Is it okay now?

**Comment by Lonami** [comment]
Right, totally forgot about the fact cryptg depends on grammers now.

I haven't released the fix yet as a new grammers version so I would wait a bit more before bumping the version here.

If we're doing this, I'm also not confident that my "fix" is enough, and would prefer to tweak it some more.

**Comment by Lonami** [comment]
Hm it's kind of annoying because I made some changes to do encryption in-place so cryptg will need to be adjusted to handle that even if I release the factorization fix. So this may take longer than I had hoped.
